bitnami/concourse/templates/web/service.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if .Values.web.enabled -}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "concourse.web.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: web
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  type: {{ .Values.web.service.type }}
+  {{- if (or (eq .Values.web.service.type "LoadBalancer") (eq .Values.web.service.type "NodePort")) }}
+  externalTrafficPolicy: {{ .Values.web.service.externalTrafficPolicy | quote }}
+  {{- end }}
+  {{ if eq .Values.web.service.type "LoadBalancer" }}
+  loadBalancerSourceRanges: {{ .Values.web.service.loadBalancerSourceRanges }}
+  {{ end }}
+  {{- if (and (eq .Values.web.service.type "LoadBalancer") (not (empty .Values.web.service.loadBalancerIP))) }}
+  loadBalancerIP: {{ .Values.web.service.loadBalancerIP }}
+  {{- end }}
+  ports:
+    - name: http
+      port: {{ .Values.web.containerPort }}
```

Comment:
The container port is different from the service port, we allow configuring both

Reply:
Example, if it is a non-root container we cannot set the containerPort to 80, but we can do it in the service